BDArmory/Control/BDModuleOrbitalAI.cs

```diff
@@ -427,7 +427,12 @@ public override void DeactivatePilot()
                 fc.Deactivate();
                 fc = null;
             }
-
+            foreach (var engine in VesselModuleRegistry.GetModuleEngines(vessel))
+            {
+                if (engine.throttleLocked || !engine.allowShutdown || !engine.allowRestart) continue; // Ignore engines that can't be throttled, shutdown, or restart
+                if (VesselSpawning.SpawnUtils.IsModularMissilePart(engine.part)) continue; // Ignore modular missile engines.
+                engine.independentThrottlePercentage = 0f; // Set all independent engines to zero throttle
```

Comment:
It'd be better to store the `independentThrottle` and `independentThrottlePercentage` for each engine when the AI activates prior to calling `UpdateEngineLists` and then restoring them to their original values when deactivating the AI (if they still exist) so that it doesn't override custom setups.